Finish "Resources" Section (#101)
Slimefun/Wiki: pages/Carbon-Chunk.md

> Carbon Chunks are [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resource)s that can be used to craft a variety of other items.

Reviewer: ```suggestion
Carbon Chunk is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resource) that can be used to craft a variety of other items.
```
